README.md

```diff
@@ -917,39 +921,89 @@ end
+#### `chewy:update`
 
-#### `chewy:deploy`
+It doesn't create indexes, it simply imports everything to the existing ones and fails if the index was not created before.
 
-This rake task is especially useful during the production deploy. Currently it executes selective reset, this means that an index will be reset only if the index specification (settings or mappings) has been changed, otherwise the reset of this index will be skipped.
+Unlike `reset` or `upgrade` tasks, it is possible to pass type references to update the particular type. In index name is passed without the type specified, it will update all the types defined for this index.
 
-Obviously at the first run it will reset everything because it needs to lock all the index specifications in the `Chewy::Stash`.
+```bash
+rake chewy:update # updates all the existing indices
+rake chewy:update[users] # updates UsersIndex only
+rake chewy:update[users,places#city] # updates the whole UsersIndex and PlacesIndex::City type
+rake chewy:update[-users,places#city] # updates every index in application except all the types defined in UsersIndex and the rest of the types defined in PlacesIndex
+```
 
-See [Chewy::Stash](lib/chewy/stash.rb) and [Chewy::Index::Specification](lib/chewy/index/specification.rb) for more details.
+#### `chewy:sync`
+
+Provides a way to synchronize outdated indexes with the source quickly and without doing a full reset.
 
-In the future, additional routines are planned during `chewy:deploy` execution. Like, additional fast or partial index updates to make sure everything is up-to-date as it was a full reset.
+Arguments are similar to the `chewy:update` ones. It is possible to specify a particular type or a whole index.
 
-Right now the approach is that if some data had been updated, but index specification had not been changed, it would be much faster to perform manual partial index update inside data migrations or even manually after the deploy.
+See [Chewy::Type::Syncer](lib/chewy/type/syncer.rb) for more details.
+
+```bash
+rake chewy:sync # synchronizes all the existing indices
+rake chewy:sync[users] # synchronizes UsersIndex only
+rake chewy:sync[users,places#city] # synchronizes the whole UsersIndex and PlacesIndex::City type
+rake chewy:sync[-users,places#city] # synchronizes every index in application except all the types defined in UsersIndex and the rest of the types defined in PlacesIndex
+```
+
+#### `chewy:deploy`
+
+This rake task is especially useful during the production deploy. It is a combination of `chewy:upgrade` and `chewy:sync` and the latter one is called only for the indexes that were not reset during the first stage.
```

Comment:
`and the latter is called only for these indices that were not reset during the first stage` (without one)